.circleci/config.yml

```diff
@@ -192,10 +97,44 @@ workflows:
             branches:
               only: master
 
-      - deploy:
+      - architect/push-to-app-collection:
+          context: architect
+          name: push-cert-exporter-to-aws-app-collection
+          app_name: "cert-exporter"
```

Comment:
I think all these jobs are missing `app_namespace` field set to monitoring. `cert-exporter` is running in monitoring namespace

Reply:
To clarify:

```suggestion
          app_name: "cert-exporter"
          app_namespace: "monitoring"
```